src/components/remotecontrol/remotecontrol.css

```diff
@@ -70,25 +50,15 @@
     left: 0;
     right: 0;
     width: 100%;
-    -webkit-box-shadow: 0 0 1.9vh #000;
     box-shadow: 0 0 1.9vh #000;
     border: .1em solid #222;
     user-drag: none;
-    user-select: none;
-    -moz-user-select: none;
-    -webkit-user-drag: none;
-    -webkit-user-select: none;
-    -ms-user-select: none
+    user-select: none
```

Comment:
This could use a semicolon, it seems to have been removed for some reason.

Reply:
There's a bunch of semicolons missing all over the place, from what I've seen. The CSS cleanup PR mostly takes care of it, but I'll add them for the lines affected by this PR.